test/controllers/board.js

```diff
@@ -19,6 +19,7 @@ describe('Board API calls', function () {
   var boardId;
 
   before(async function (done) {
+    this.timeout(5000); //to await the email server process
```

Comment:
I mean its not a pause, is only more time than the default (2000) because email server needs more than 2seg to send the email with the URL and return the success. if I see the time counter of the case it takes lees than 3000ms ¿im wrong?